src/ocean/utils/WebServiceConnector.ts

```diff
@@ -1,11 +1,12 @@
-import { BodyInit, RequestInit, Response } from 'node-fetch'
+import { BodyInit, RequestInit, Response } from 'cross-fetch/lib.fetch'
 import fs from 'fs'
 import { Logger } from '../../utils'
 import save from 'save-file'
 // import { createWriteStream } from 'streamsaver'
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const fetch = require('node-fetch')
+const fetch = require('cross-fetch')
```

Comment:
can't recall the exact reason why we specifically did a `require` here, but it was done a long time ago in squid.js. So we can go back to our used `import` pattern. So this should be instead:

```js
import { fetch } from 'cross-fetch'
```

Same for the `timeout-signal` import

Reply:
Ok sure

Reply:
The `import` pattern doesn't work for `timeoutSignal`:
 `'timeoutSignal' can only be imported by using 'import timeoutSignal = require("timeout-signal")' or a default import.`

Reply:
> ...or a default import.

yup, let's pick that :-) Library has only one default export so can only be used with a default import (instead of single import with `{ }`):

```js
import timeoutSignal from 'timeout-signal'
```

Reply:
The `timeoutSignal` is working fine as a default import. However,  when I used `import fetch from 'cross-fetch'` it gave me the following error:
```
Type 'Response' is missing the following properties from type 'Response': buffer, size, textConverted, timeout
```

I have changed it on the branch now to `const fetch = require('cross-fetch')` and it's currently working & passing the tests.

Reply:
well, we might have found a reason for that `require` usage here, seems to be the only way to get typings right without having to refactor the `WebServiceConnector` or further investigate why the typings between node-fetch and cross-fetch are so different

Reply:
Yeah, it seems like it